src/components/Attachments/AttachmentCarousel/Pager/index.tsx

```diff
@@ -70,13 +73,27 @@ function AttachmentCarouselPager(
         isPagerScrolling.value = e.offset !== 0;
     }, []);
 
+    useAnimatedReaction(
+        () => isPagerScrolling.value,
+        (isScrolling, previousIsScrolling) => {
+            if (isScrolling === previousIsScrolling) {
+                return;
+            }
+            runOnJS(onIsPagerScrollingChange ?? (() => {}))(isScrolling);
+        },
+        [onIsPagerScrollingChange],
+    );
+
```

Comment:
@wildan-m Why is this change required?

Reply:
We need to adjust the scrolling state so it can be utilized in AttachmentModal

On iOS, tapping the carousel arrow currently causes a crash as it simultaneously toggles the highRes caption

The explanation is here:

https://github.com/Expensify/App/blob/4bf56529ab98f2c7725a09baeba3f8bb668f9095/src/components/AttachmentModal.tsx#L204